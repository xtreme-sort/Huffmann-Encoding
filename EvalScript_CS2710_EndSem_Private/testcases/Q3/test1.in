GenerateTreeEnhanced
63
f 1
n 1
p 3
s 3
9 6
_ 6
V 7
4 8
G 8
l 8
Q 10
S 10
W 10
q 10
m 11
0 12
B 12
J 12
b 12
t 14
u 14
T 16
z 17
L 19
a 19
M 21
Y 22
F 23
c 24
h 24
D 26
7 28
I 28
O 28
R 28
3 29
y 29
C 30
P 30
N 32
d 35
v 35
8 36
i 37
w 37
2 38
A 39
U 39
k 39
5 42
g 42
j 43
1 46
6 46
H 46
r 46
K 47
x 47
E 48
e 48
X 50
Z 50
o 50
Quit
