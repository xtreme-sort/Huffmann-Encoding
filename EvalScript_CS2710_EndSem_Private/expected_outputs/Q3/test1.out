(((((A,U),(k,((S,W),M))),((5,g),(j,((q,m),Y)))),(((1,6),(H,r)),((K,x),((F,c),E)))),((((e,(h,(0,B))),(X,Z)),((o,((J,b),D)),((((9,_),t),7),(I,O)))),((((R,3),(y,(u,(V,4)))),((C,P),(N,(T,(G,l))))),(((d,v),((z,((s,((f,n),p)),Q)),8)),((i,w),(2,(L,a)))))));
